### DISCUSSION TITLE
Windows NT 3.1 build 340 Installation Problem.

### DISCUSSION THREAD

**Discussion by agajgjsa592586** [86Box/86Box#4894] [General]
Hey guys, I was trying to install Windows NT 3.1 build 340 on 86Box 4.0 (I think it's old). It gives me the error that I can't install it on a current processor. I tried 386, but no dice. I gave up there for my life. I needed help from you! Do you know what other options are to install successfully?

**Comment by TC1995** [comment]
Just try the nightly builds.

**Comment by agajgjsa592586** [comment]
It didn't work with Intel Pentium or higher. And I'm sad to do this in an other way...

**Reply by agajgjsa592586** [reply]
However, I needed to find a way to make the installation confuse itself somehow.

**Comment by agajgjsa592586** [comment]
I feel like only 386 work. Not a bug, but it is still annoying.
